package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -259,8 +267,20 @@ def write_next_timestep(self, ts=None):
         else:
             coordinates = ts.positions
 
+        # Write number of atoms
         self._xyz.write("{0:d}\n".format(ts.n_atoms))
-        self._xyz.write("frame {0}\n".format(ts.frame))
+
+        # Write remark
+        if self.remark is None:
+            remark = "Written by MDAnalysis {0} (release {1})".format(
+                self.__class__.__name__, __version__
+                ) + " | Frame {0}\n".format(ts.frame)
```

Comment:
Might people who write out .xyz trajectories be caught off-guard by this remark if they used to rely on reading the frame? `Frame {0} | Written by ...` might work better here -- and the more dynamic information is then shown first.

Reply:
Good point! =)